### ISSUE TITLE
CI Tests fail 

### ISSUE THREAD

**Issue by alperyeg** [NeuralEnsemble/elephant#33]
When doing the PR in https://github.com/NeuralEnsemble/elephant/pull/32 I noticed the nosetests performed by Travis CI fail, with following error:
```python
AssertionError: attr is not equal [names]: FrozenList([None]) != FrozenList([u'0'])
```

I checked and saw that this issue is related to the `test_pandas_bridge.py` unittest. It seems that the `panda` version specified in the install script has problems with the newest `numpy` library or vice versa. 
However after trying locally with all the newest versions the tests still fail. 
I assume the unittest regarding the panda test has to be adapted according to the newest versions.

**Comment by alperyeg** [comment]
@toddrjen any clues on this?

**Comment by apdavison** [comment]
fixed by #46
